tools/pg_diff

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env node
+
+const async = require('async');
+const ERR = require('async-stacktrace');
+const colors = require('colors');
+const fs = require('fs-extra');
+const _ = require('lodash');
+const path = require('path');
+
+const databaseDescribe = require('../tests/databaseDescribe');
+const databaseDiff = require('../tests/databaseDiff');
+
+const yargs = require('yargs')
+    .usage('Usage: $0 [options]')
+    .describe('db', 'reads a description from the named database')
+    .describe('dir', 'read a description from a directory that\'s been generated by pg_describe')
+    .describe('ignore-tables', 'a list of tables to ignore')
+    .array('ignore-tables')
+    .describe('ignore-enums', 'a list of enums to ignore')
+    .array('ignore-enums')
+    .help('h')
```

Comment:
Could we also add an `--ignore-column` argument? This could look like `--ignore-column users.receive_email` which would mean to not print anything about differences in the `receive_email` column of the `users` table.